libs/libc/Makefile

```diff
@@ -194,9 +196,9 @@ distclean:: clean
 	$(Q) $(MAKE) -C kbin distclean
 	$(Q) $(MAKE) -C zoneinfo distclean BIN=$(BIN)
 	$(call DELFILE, exec_symtab.c)
-	$(call DELFILE, bin/Make.dep)
-	$(call DELFILE, kbin/Make.dep)
+	$(call DELFILE, bin$(DELIM)Make.dep)
```

Comment:
why mix / and $(DELIM)

Reply:
DELFILE use `del` which is a pure windows command, must use Backslash, so i change 
$`(call DELFILE, bin/Make.dep)` to `$(call DELFILE, bin$(DELIM)Make.dep)`